website/docs/documentation/Pipelines/Smart-seq2_Single_Nucleus_Pipeline/README.md

```diff
@@ -0,0 +1,136 @@
+
+| Reference name | Reference Description | Type |
+| --- | --- | --- |
+| tar_star_reference | Genome references for STAR alignment | TAR |
+| annotations_gtf | Custom GTF file containing annotations for exon and intron tagging; must match the STAR reference | GTF | 
+| genome_ref_fasta | FASTA file used for STAR alignment | FASTA | 
+| stranded |  Library strand information; values are FR, RF, or NONE | String | 
+| input_id | Unique identifier string for the cell; can be a UUID or human-readable ID | String |
+| input_id_metadata_field | Optional value that can be used to identify any metadata fields specific to consortia (BRAIN Initiative, HCA, etc. ) that were used for the input_id | String |
+| input_name | Optional unique identifier string for the cell; can be a human-readable alternative if a UUID is used for the input_id | String |
+| input_name_metadata_field | Optional value that can be used to identify any metadata fields particular to consortia (e.g. Brain Initiative, HCA, etc. ) that were used for the input_name  | String |
+| output_name | Unique name used for output files; can also be a cloud path | String |
+| adapter_list | File listing adaptor sequences used in the library preparation (i.e. Illumina adaptors for Illumina sequencing) | File | 
+| paired_end | Boolean description for if samples are paired-end | Boolean |
+
+
+## Running Smart-seq2
+
+The [snSS2 workflow ](https://github.com/broadinstitute/warp/tree/develop/pipelines/skylab/smartseq2_single_nucleus/SmartSeq2SingleNucleus.wdl) is in the [pipelines/smartseq2_single_nucleus folder](https://github.com/broadinstitute/warp/tree/snSS2_first_wdls/pipelines/skylab/smartseq2_single_nucleus) of the WARP repository and implements the workflow by importing individual tasks (written in WDL script) from the WARP [tasks folder](https://github.com/broadinstitute/warp/tree/master/tasks/skylab/).
+
+### Smart-seq2 Single Nucleus Workflow Summary
+| Task name and task’s WDL link | Description | Software | Tool |
+| --- | --- | --- | --- |
+| [TrimAdapters.TrimAdapters](https://github.com/broadinstitute/warp/tree/develop/tasks/skylab/TrimAdapters.wdl) | Trims adaptor sequences from the FASTQ inputs | [ea-utils](https://github.com/ExpressionAnalysis/ea-utils) | [fastq-mcf](https://github.com/ExpressionAnalysis/ea-utils/tree/master/clipper) |
+| [StarAlignFastq.StarAlignFastqPairedEnd](https://github.com/broadinstitute/warp/tree/develop/tasks/skylab/StarAlign.wdl) | Aligns reads to the genome | STAR | 
+| [Picard.RemoveDuplicatesFromBam](https://github.com/broadinstitute/warp/tree/develop/tasks/skylab/Picard.wdl) | Removes duplicate reads, producing a new BAM output; adds regroups to deduplicated BAM | Picard | MarkDuplicates, AddOrReplaceReadGroups |
```

Comment:
I'm curious why we're removing duplicate reads instead of just marking them as duplicates and ignoring them downstream? @kishorikonwar @khajoue2 Typically we don't throw away any data in our pipelines.

Reply:
We wanted to stay consistent with Allen's pipeline. Featurecounts also may not handle duplicates correctly if we do't remove them.

Reply:
Ok, I've made a ticket to possibly look into this in the future. I prefer when we keep all of the data and just provide filtering suggestions. Not considered high enough priority to look into before releasing the pipeline.